geode-experimental-driver/src/integrationTest/java/org/apache/geode/experimental/driver/SSLTest.java

```diff
@@ -179,7 +179,7 @@ public void driverCanConnectWithOneWayAuthentication() throws Exception {
 
   @Test
   public void driverCannotConnectIfProtocolsMismatch() throws Exception {
-    startLocator(SERVER_KEY_STORE, SERVER_TRUST_STORE, true, "TLSv1.1", "any");
+    startLocator(SERVER_KEY_STORE, SERVER_TRUST_STORE, true, "TLSv1.3", "any");
```

Comment:
Is this because of TLS version change in the product? Does the test needs to be changed in future if there are version changes in the future?